feat(utxo): add new fixed txfee option for DINGO-like coins

dimxy opened the pull request:
Before this PR, for the utxo fixed fee calculation we rounded up txsize up to 1000 bytes. It was once required for the DOGE coin (but not anymore), but we did this for all utxo coins (what in fact unnecessarily increased txfee).
This rounding up is now required for few coins, like DINGO or RUNES.

With this PR, this txsize rounding up is not done by default.
Added a new fixed tx fee option `"dingo_fee"=true`, with rounding up, that could be set for a coin in the coins file.

Fixes #2446 issue.

@cipig

cipig commented:
![image](https://github.com/user-attachments/assets/253d6ad9-721f-47fb-980e-cf5f5c8d0fc4)
RUNES works fine now
DINGO with param `"dingo_fee": true` in coins file too

cipig commented:
can i please have a new build of this? the old one was removed

dimxy commented:
Merged with dev and fixed several preimage volume tests broken due to new DEFAULT_SWAP_TX_SPEND_SIZE value in the dev (https://github.com/KomodoPlatform/komodo-defi-framework/pull/2454/commits/ad1cdf4db8e5649fedf2ade0128a0987a084eb05).

(Note: I added env vars _KDF_NO_UTXO_DOCKER, _KDF_NO_QTUM_DOCKER etc to disable start of unneeded docker containers for easier docker tests debugging)

shamardy commented:
> can we please merge this? using it on all makers since a long time and works perfectly fine

Have to do my own line by line review, utxo fee changes PRs are usually complex and require rigorous review.

shamardy reviewed (dismissed):
LGTM!

shamardy commented:
@dimxy `test_electrum_tx_history` fails here and not in dev

dimxy commented:
Thank you
(I wanted yet to check why this test was failing: test_order_cancellation_received_before_creation. I ll fix it in another PR then, if possible to fix)

shamardy commented:
> I wanted yet to check why this test was failing: test_order_cancellation_received_before_creation. I ll fix it in another PR then, if possible to fix

It's failing in dev so it's not this PR. I did check it yesterday and went back all the commits until the commit that introduced it and it still fails locally. I am thinking of ignoring it or changing how the test covers this case.

dimxy commented:
> > I wanted yet to check why this test was failing: test_order_cancellation_received_before_creation. I ll fix it in another PR then, if possible to fix
> 
> It's failing in dev so it's not this PR. I did check it yesterday and went back all the commits until the commit that introduced it and it still fails locally. I am thinking of ignoring it or changing how the test covers this case.

Yeah I thought maybe just increasing the wait-for-log timeout would help but it didn't, seems a more complex problem.
